src/Sulu/Bundle/CoreBundle/Command/ServerRunCommand.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\CoreBundle\Command;
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class ServerRunCommand extends \Symfony\Bundle\FrameworkBundle\Command\ServerRunCommand
```

Comment:
Usually we import these classes as `BaseServerRunCommand`, so that this line is shorter.